scripts/ci/lib.sh

```diff
@@ -859,6 +859,27 @@ handle_nightly_runs() {
     fi
 }
 
+handle_nightly_roxctl_mismatch() {
+    if ! is_OPENSHIFT_CI; then
+        die "Only for OpenShift CI"
+    fi
+
+    if is_in_PR_context || ! [[ "${JOB_NAME_SAFE:-}" =~ ^nightly- ]]; then
```

Comment:
how come this one is `^nightly-` but we only check for `nightly` in `is_nightly_run`?

Reply:
JOB_NAME_SAFE comes from the test name in openshift/release.
CIRCLE_TAG is how we 'tag' nightlies e.g. 3.70.x-nightly-20220530